hawc/templates/includes/paginator.html

```diff
@@ -26,5 +26,16 @@
         <a class="page-link" href="#">&gt;&gt;</a>
     </li>
     {% endif %}
+
+    <li class="ml-3 page-item">
```

Comment:
If we add class `disabled` we can get rid of the hover styles over the label, so that users don't mistake the label as an interactive element.

Reply:
Yes, this was intentional. I wanted people to know that that it is interactive; if they click the goto label, it focuses the page select. This is consistent with how other labels in forms work. Ok to leave as is?

Reply:
Sure, makes sense!